Documentation/kubernetes-on-baremetal.md

```diff
@@ -38,12 +38,18 @@ Install CoreOS using the bare metal installation instructions:
 
 Mixing multiple methods is possible. For example, doing an install to disk for the machines running the etcd cluster and Kubernetes master nodes, but PXE-booting the worker machines.
 
+
 [coreos-ipxe]: https://coreos.com/os/docs/latest/booting-with-ipxe.html
 [coreos-pxe]: https://coreos.com/os/docs/latest/booting-with-pxe.html
 [coreos-ondisk]: https://coreos.com/os/docs/latest/installing-to-disk.html
+[etcd-cluster-example]: https://github.com/coreos/matchbox/blob/master/Documentation/getting-started-rkt.md
+[ignition-docs]: https://coreos.com/ignition/docs/latest/
+[matchbox-gh]: https://github.com/coreos/matchbox
+[matchbox-intro-doc]: https://github.com/coreos/matchbox/blob/master/Documentation/matchbox.md
+[multi-node-example]: https://github.com/coreos/matchbox/blob/master/Documentation/kubernetes.md
+[self-hosted-example]: https://github.com/coreos/matchbox/blob/master/Documentation/bootkube.md
```

Comment:
These are kinda in the automated section, but I'm wondering if maybe we just want to collapse these down and have a section called "Provisioning" keeping the description of matchbox and having all these links. Since they're all using PXE and Ignition